### PR TITLE
[ZH] Translate 14-1.md

### PR DESCRIPTION
**Pull request by titusjgr**

### CODE REVIEW

File: docs/zh/week14/14-1.md [hunk 1]
```diff
@@ -0,0 +1,288 @@
+
```
Comment (Atcold) [inline docs/zh/week14/14-1.md:1]:
```suggestion

```

File: docs/zh/week14/14-1.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+---
+lang-ref: ch.14-1
```
Comment (Atcold) [inline docs/zh/week14/14-1.md:3]:
```suggestion
lang-ref: ch.14-1
lang: zh
```

**Review by Atcold — CHANGES_REQUESTED** [review]
You need to update `docs/_config.yml` as well.

**Comment by titusjgr** [comment]
I made the changes accordingly.

**Review by Atcold — APPROVED** [review]
Thanks!

**Pull request merged**